experimental/handoff-library/csharp_dotnetcore/samples/LPProxyBot/Bots/LPProxyBot.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using LivePersonConnector;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json.Linq;
+
+namespace LPProxyBot.Bots
+{
+    public class LPProxyBot : ActivityHandler
```

Comment:
I am not sure I love the name. LivePersonProxyBot sounds way more descriptive. Thoughts?

Reply:
Agree. Fixing this and other abbreviations.